modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -1693,4 +1697,144 @@ test.describe('Manage object entries through Workflow', () => {
+				{
+					apiHelpers,
+					objectFieldBusinessTypes: ['dateTime'],
+				}
+			);
+
+			const objectDefinitionAPIClient =
+				await apiHelpers.buildRestClient(ObjectDefinitionAPI);
+
+			const {body: objectDefinition} =
+				await objectDefinitionAPIClient.postObjectDefinition({
+					active: true,
+					enableLocalization: true,
+					label: {
+						en_US: objectDefinitionLabel,
+					},
+					name: objectDefinitionName,
+					objectFields,
+					pluralLabel: {
+						en_US: objectDefinitionLabel,
+					},
+					portlet: true,
+					scope: 'company',
+					status: {
+						code: 0,
+					},
+					titleObjectFieldName,
+				});
+
+			apiHelpers.data.push({
+				id: objectDefinition.id,
+				type: 'objectDefinition',
+			});
+
+			await viewObjectEntriesPage.goto(objectDefinition.className);
+
+			// Create object entry date time
+
+			await viewObjectEntriesPage.addObjectEntryButton.click();
+
+			await viewObjectEntriesPage.dateTimeInput.fill(
+				'10/05/2025 12:00 PM'
+			);
+
+			await viewObjectEntriesPage.saveObjectEntryButton.click();
+
+			await expect(viewObjectEntriesPage.successMessage).toBeVisible();
+
+			// Create user with permissions
+
+			const company =
+				await apiHelpers.jsonWebServicesCompany.getCompanyByWebId(
+					'liferay.com'
+				);
+
+			const user = await createUserWithPermissions({
+				apiHelpers,
+				rolePermissions: [
+					{
+						actionIds: ['VIEW_CONTROL_PANEL'],
+						primaryKey: company.companyId,
+						resourceName: '90',
+						scope: 1,
+					},
+					{
+						actionIds: ['ACCESS_IN_CONTROL_PANEL'],
+						primaryKey: company.companyId,
+						resourceName:
+							'com_liferay_users_admin_web_portlet_UsersAdminPortlet',
+						scope: 1,
+					},
+					{
+						actionIds: ['ACCESS_IN_CONTROL_PANEL'],
+						primaryKey: company.companyId,
+						resourceName: `com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectDefinition.className.split('#')[1]}`,
+						scope: 1,
+					},
+					{
+						actionIds: ['VIEW'],
+						primaryKey: company.companyId,
+						resourceName: `${objectDefinition.className}`,
+						scope: 1,
+					},
+				],
+			});
+
+			// Switch to created user
+
+			await performUserSwitch(page, user.alternateName);
+
+			// Change user timezone and check if the time has changed
+
+			await usersAndOrganizationsPage.changeUserTimezone(
+				'America/Sao_Paulo',
+				user.alternateName
+			);
+
+			await viewObjectEntriesPage.goToDateTimeEntry(
+				objectDefinition.className
+			);
+
+			await expect(viewObjectEntriesPage.dateTimeBR).toHaveValue(
+				'10/05/2025 09:00 AM'
+			);
+
+			// Change user timezone and check if the time has changed
+
+			await usersAndOrganizationsPage.changeUserTimezone(
+				'UTC',
+				user.alternateName
+			);
+
+			await viewObjectEntriesPage.goToDateTimeEntry(
+				objectDefinition.className
+			);
+
+			await expect(viewObjectEntriesPage.dateTimeUTC).toHaveValue(
+				'10/05/2025 12:00 PM'
+			);
```

Comment:
this part is not necessary, since the raw value of the entry is `10/05/2025 12:00 PM` and you already saw that on the different user is `10/05/2025 09:00 AM`. i saw the poshi and it tested only once too, please let me know if i got it wrong.

Reply:
if this part is not necessary anymore and you could find a way to put the timezone directly when creating the user using the api, there would be no need for the `changeUserTimezone` function anymore